proc_chirp.py

```diff
@@ -1,6 +1,5 @@
 from pyspecdata import *
-#import logging
-#init_logging(level=logging.DEBUG)
+#init_logging(lebel='debug')
 #{{{ boolean statement for processing data before modification to generate chirp
 #   that sets voltage of ref (CH1 of scope) and DUT (CH2 of scope) to same value
 corrected_volt = True
```

Comment:
I need to double check with alec but I am fairly certain this is an outdated version of proc_square_refl?

Reply:
ok, then you should probably use the updated version

Reply:
(add to repo + push)

Reply:
the updated version is proc_square_refl

Reply:
if it's really that the one is newer and the other is older, keep only the newer

do keep in mind that you want to be able to process both square and chirp pulses

Reply:
so according to alec this processes chirp pulses whereas the other processes square wave pulses. I will implement the runtime error as I have not used this code